src/logic/transferTracker.ts

```diff
@@ -0,0 +1,43 @@
+import { EnrichedTransaction } from "helius-sdk";
+import { transactionStatuses } from '@/app/types/splTransfer';
+import { addTransactionStatus, updateSplTransfer } from "@/services/db/queries/splTransfer";
+import bs58 from 'bs58';
+
+/**
+ * Updates the transfer database record with details from a Helius enriched transaction.
+ * @param {EnrichedTransaction} enrichedTransaction - The enriched transaction from Helius.
+ * @returns {Promise<void>}
+ */
+export async function updateTransferDetails(enrichedTransaction: EnrichedTransaction): Promise<void> {
+  try {
+    const { signature, instructions, slot, timestamp, fee } = enrichedTransaction;
+
+    const memoIx = instructions.find((instruction) => {
+      return instruction.programId.toString() === 'MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr';
```

Comment:
Noticed this address in a few places. Consider making a const.
Unlikely to change, but Token/Token22 sets precedent.